cumulusci/tasks/release_notes/task.py

```diff
@@ -1,6 +1,24 @@
+# -*- coding: utf-8 -*-
+"""Release Note Tasks
+
+Classes:
+    GithubReleaseNotes
+    ParentPullRequestNotes
```

Comment:
I'd leave this out. It's liable to become out of date compared to something like the VSCode Outline panel that is actually aware of the code.